### PR TITLE
Updated Readme.md with the doc file provided

### LINKED ISSUE #200: Transcribe .doc file to Readme.md file
### Explication

We have the project documentation at one Google Doc File. We need to move this documentation file to the Readme.md file to this can looks equals to google doc file. 

Please check at this [link](https://docs.google.com/document/d/1TGESSinCKMWw-BMkWO0kiJlu8jqZjXxJ4ldqt5nVZAQ/edit) the Google Doc File

### Screenshots

imagen

### Checklist

This issue can be closed when the following tasks are complete:

- [ ] The .doc file has been transcribed to markdown format
- [ ] The Readme.md content now is the content that is at google doc file
- [ ] The Readme.md index has been updated to new sections at the file
- [ ] The Readme.md preserve the language badges, index and image of project
- [ ] The Readme.md have authors and co authors present at the Collaborators section of this project 

### References

https://github.com/Azordev/sencico/blob/dev/README.md
https://github.com/Azordev/sencico/blob/dev/README.md#colaboradores

### How to do

1. Open the google doc documentation https://docs.google.com/document/d/1TGESSinCKMWw-BMkWO0kiJlu8jqZjXxJ4ldqt5nVZAQ/edit
2. Clone the repo
3. Change the branch to fix/#200
3. Enter at Readme.md
4. Change the needed. U can use an extension of VSCode to check at preview of markdown files. This is an example: https://code.visualstudio.com/api/extension-guides/markdown-extension

### PR DESCRIPTION
**Pull request by heliomar-pena**
### Description

I have updated the readme.md with the changes proposed by JHernandez @elpollo12 in her Google Doc file. I have transcribe this to a markdown file and have updated the image to imgur to can be used to the project. I have used HTML to do that the image was be aligned to the left / right with the text

fixes #200 

#### Screenshots

imagen

#### Type of change

- [x] New feature (non-breaking change which adds functionality)

### How to test it

1. Go to here to check preview: https://github.com/Azordev/dasher-user/blob/c05e0d5f93caa08a8a57ffa378b8a2ff509dd85e/README.md

### Checklist:

- [x] The .doc file has been transcribed to markdown format
- [x] The Readme.md content now is the content that is at google doc file
- [x] The Readme.md index has been updated to new sections at the file
- [x] The Readme.md preserve the language badges, index and image of project
- [x] The Readme.md have authors and co authors present at the Collaborators section of this project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,152 +40,174 @@ Do you want to know how your shipment is going and how long it will arrive? You
...
-Feel free to fork this project and improve it!
+**How do I create a new component?**
 
+In the component folder, create a new file with the .jsx extension. Make sure to put an appropriate name to the functionality that you are going to develop.
+
+## Collaborators
+
+| [Israel Laguan][author-github] | ![email-icon][] Email me to [contact@israellaguan.com][author-email] / ![linkedin-icon][] Connect to [my Linkedin][author-linkedin] |
+| :----------------------------: | :---------------------------------------------------------------------------------------------------------------------------------: |
+|        ![author-pic][]         |                                                             ![banner][]                                                             |
```
Comment (Israel-Laguan) [inline README.md]: Please remove banner, seems like it breaks for others, I prefer everyone looks similar instead of stands out

**Comment by Israel-Laguan** [comment]
We can use Github as image repository, just create a folder named `docs` or similar and put the images inside a folder `assets` or similar, to not rely on third party sites

File: README.md (hunk 1 above)
Comment (heliomar-pena) [reply README.md]: It would be nice to have a single "azordev" Banner. What do you think?

**Comment by Israel-Laguan** [comment]
I think its not necessary that you align to left or right, please try a little bit more, and if its hard you have 2 options:

- use a table to align. Pro: it will work. Con: will looks weird
- just put image as normal, no alignment. Pro: easy and painless. Con: it looks different that source.

**Review by e-azocar — APPROVED** [review]

**Review by Israel-Laguan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Israel-Laguan) [reply README.md]: Lets include this in a new issue

**Pull request merged**